providers/common/securitycheck_default.c

```diff
@@ -28,6 +28,12 @@ int ossl_tls1_prf_ems_check_enabled(OSSL_LIB_CTX *libctx)
     return 0;
 }
 
+/* Disable the short MAC length check in the default provider */
+int ossl_no_short_mac_enabled(OSSL_LIB_CTX *libctx)
```

Comment:
just use the FIPS_ function and get rid of this. It has no purpose now that we have to do #ifdef FIPS_MODULE where we use it.

Reply:
At some point after everything is merged from lots of PR's we should probably change the FIPS_ name to an ossl_ name